browse/templates/abs/extra_services.html

```diff
@@ -189,11 +185,14 @@ <h2>Access Paper:</h2>
       <div class="abs-license">
         {%- if withdrawn -%}
         <div hidden>No license for this version due to withdrawn</div>
-        {%- elif abs_meta.license.icon_uri_path -%}
-        <a href="{{abs_meta.license.effective_uri}}" title="Rights to this article"><img src="https://arxiv.org{{ abs_meta.license.icon_uri_path }}"/></a>
         {%- else -%}
-        (<a href="{{abs_meta.license.effective_uri}}" title="Rights to this article">view license</a>)
+          {%- if 'other' in formats -%}
```

Comment:
This code is not in master or develop and doesn't seem to have anything to do with HTML-papers.